css/mystyle.css

```diff
@@ -207,6 +206,31 @@ color: #838994;
 	text-align: center;
 
 }
+.clear{
+	clear: both;
+}
+
+@media only screen and (max-width: 1010px) {
+	.assign .illustration > li {
+		float: none;
+		width: 100%;
+	}
+}
+
+@media only screen and (max-width: 630px) {
+	#name {
+		position: relative;
+		text-align: center;
+	}
+	.top-navigation {
```

Comment:
Makes the top area taller to allow more room for the name and navigation to be on top of each other.